project/project.mjs

```diff
@@ -1,12 +1,12 @@
-/** Logic for the /project endpoint */
 import * as utils from '../utilities/shared.mjs'
+import dbDriver from '../database/driver.mjs'
 import * as fs from 'fs'
 
+const database = new dbDriver()
```

Comment:
This is never used and can be removed

Reply:
Thanks I have removed it and updated in the latest code .